src/createThemeFile.ts

```diff
@@ -30,6 +30,7 @@ function createThemeFile({
 	}
 
 	const CSSModulesData = themedWidgets.map(({ themeKey, fileName }) => {
+		themeKey = themeKey.split(/[\\]+/).join('/');
```

Comment:
This is fine, we have used [slash from npm](https://www.npmjs.com/package/slash) before in other packages.